src/fragment-components/a-inline-loading.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import {
+	RadioButtonGroup,
+	RadioButton,
+	InlineLoading,
+	TextInput
+} from 'carbon-components-react';
+import { AComponent } from './a-component';
+import { ComponentInfo } from '.';
+import { css } from 'emotion';
+import image from './../assets/component-icons/inline-loading.svg';
+import {
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj,
+	angularClassNamesFromComponentObj
+} from '../utils/fragment-tools';
+
+export const AInlineLoadingSettingsUI = ({ selectedComponent, setComponent }: any) => {
+	return <>
+		<legend className="bx--label">Status</legend>
```

Comment:
Use single quotes everywhere. I think the only place we use double is in exports and that's because of wrapping quotes in quotes.